src/main.rs

```diff
@@ -155,7 +155,7 @@ fn create_app<'a>(
             Arg::new("platform")
                 .long("platform")
                 .help(
-                    "num_pci_segments=<num pci segments>,iommu_segments=<list_of_segments>,serial_number=<(DMI) device serial number>",
+                    "num_pci_segments=<num pci segments>,iommu_segments=<list_of_segments>,serial_number=<(DMI) device serial number>,uuid=<(DMI) device uuid>",
```

Comment:
Because it's in the struct it can be serialised/deserialised in the API. It would be a nice idea to update `vmm/src/api/openapi.yaml` which is used by some projects for API glue code autogeneration.

Reply:
> For the sake of consistency, would you consider changing this to `serial_number=<dmi_device_serial_number,uuid>=<dmi_device_uuid>`
OK, I will modify it.
> The two more important questions to me:
> 
> * Is the DMI device serial number and uuid exported via the rest interface by the implementation?

Serial number and uuid are set on the command line, or they are can be set via ch-remote，like "platform":{"uuid":1e8aa28a-435d-4027-87f4-40dceff1fa0a}.
I will add uuid to vmm/src/api/openapi.yaml.

> * Are these two options added to the platform struct?

Reply:
> For the sake of consistency, would you consider changing this to `serial_number=<dmi_device_serial_number,uuid>=<dmi_device_uuid>`

OK, I will modify it.

> 
> The two more important questions to me:
> 
> * Is the DMI device serial number and uuid exported via the rest interface by the implementation?

Serial number and uuid are set on the command line, or they are can be set via ch-remote，like "platform":{"uuid":1e8aa28a-435d-4027-87f4-40dceff1fa0a}.
I will add uuid to vmm/src/api/openapi.yaml.

> * Are these two options added to the platform struct?

Reply:
@Lxiaoyouling please note I typed this review earlier in my phone. As such, there are some typos in the suggested command line consistency change.